mods/ts/rules/civilian-infantry.yaml

```diff
@@ -270,13 +272,16 @@ VISC_LRG:
 		HP: 500
 	Armament:
 		Weapon: SlimeAttack
+		FireDelay: 10
```

Comment:
They do indeed attack on your replay, but still not when I play.  I must be doing something wrong...?  Here's a replay where I chase around and then very slowly kill an adult visceroid with a buggy without retaliation: https://www.dropbox.com/s/my2kaw29op4hthu/OpenRA-2017-01-15T202124Z.orarep?dl=0